recipes/anvio-minimal/meta.yaml

```diff
@@ -59,5 +53,10 @@ about:
   home: http://merenlab.org/software/anvio/index.html
   license: GPL-3.0-or-later
   license_family: GPL3
+  license_file: LICENSE.txt
   summary: "An interactive analysis and visualization platform for omics data"
   dev_url:  https://github.com/merenlab/anvio
+
+extra:
+  skip-lints:
+    - should_be_noarch_python
```

Comment:
If we don't have a compiler, it seems you have a noarch version. Is that not the case?